### PR TITLE
feat: add resizeMode prop to the Video component and update native_handlers docs

### PR DESCRIPTION
**Pull request by khushal87**
## 🎯 Goal

This PR adds the resizeMode support to the native Video component. Also, the native handlers guide is updated.

## 🛠 Implementation details

The prop support is added, and the docs are updated for usage.

## 🎨 UI Changes

<details>
<summary>iOS</summary>

<table>
    <thead>
        <tr>
            <td>Before</td>
            <td>After</td>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>
                
            </td>
            <td>
                
            </td>
        </tr>
    </tbody>
</table>
</details>

<details>
<summary>Android</summary>

<table>
    <thead>
        <tr>
            <td>Before</td>
            <td>After</td>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>
                
            </td>
            <td>
                
            </td>
        </tr>
    </tbody>
</table>
</details>

## ☑️ Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- [x] Documentation is updated
- [x] New code is tested in main example apps, including all possible scenarios
  - [x] SampleApp iOS and Android
  - [x] Expo iOS and Android

### CODE REVIEW

File: package/expo-package/src/handlers/Video.tsx [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import { VideoComponent } from '../optionalDependencies/Video';
 
 export const Video = VideoComponent
-  ? ({ onPlaybackStatusUpdate, paused, style, uri, videoRef }) => (
+  ? ({ onPlaybackStatusUpdate, paused, resizeMode, style, uri, videoRef }) => (
       <VideoComponent
         onPlaybackStatusUpdate={onPlaybackStatusUpdate}
         ref={videoRef}
-        resizeMode='contain'
+        resizeMode={resizeMode}
```
Comment (vanGalilea) [inline package/expo-package/src/handlers/Video.tsx:8]: What does it default to?

Comment (khushal87) [reply package/expo-package/src/handlers/Video.tsx:8]: `contain`

Check https://github.com/GetStream/stream-chat-react-native/blob/develop/package/src/components/ImageGallery/components/AnimatedGalleryVideo.tsx#L198

**Review by vanGalilea — APPROVED** [review]

**Comment by vanGalilea** [comment]
Fiex #1877

**Pull request merged**